crates/biome_yaml_parser/tests/yaml_test_suite/ok/block/mapping/simple_block_mapping.yaml

```
@@ -1,3 +1,4 @@
 a: 10
 'b': [20]
 "c": {30}
+*alias: 40
```

YAMLlint (1.38.0)

[error] found undeclared alias "alias"

(anchors)